HOWTO-HACK.txt

```diff
@@ -2,54 +2,74 @@ Instructions for folks who want to contribute to processing.py itself.
 
 Prerequisites:
 
-	git - http://git-scm.com/downloads
-	The latest JDK - http://www.oracle.com/technetwork/java/javase/downloads/index.html
-	Apache ant - http://ant.apache.org/bindownload.cgi
+    git - http://git-scm.com/downloads
+    JDK 8u144 - http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html
```

Comment:
I don't want to have to update this file every time Processing changes their build requirement. Did you find that the "latest" didn't work?

Reply:
Yeah, I couldn't get the JDK 9 line to work, the new line that that page references. It failed like this during the build:

<img width="1232" alt="ant_build_error_jdk" src="https://user-images.githubusercontent.com/57281/30775731-908e0be4-a067-11e7-9c8d-415be29d9632.png">

I assume it's non-trivial for the processing team to upgrade to the new major line of the JDK, being tied to JDK 8, or more specifically, 8u144, declared by the error message above and the "latest" release of JDK 8.

I don't think the build requirement, if for JDK 8, is much of a moving target, but perhaps it's sufficient to say "latest of the JDK 8?"

Reply:
That would be great; thanks.